plugins/modules/iam_server_certificate.py

```diff
@@ -231,16 +237,31 @@ def load_data(cert, key, cert_chain):
     if cert and os.path.isfile(cert):
         with open(cert, 'r') as cert_fh:
             cert = cert_fh.read().rstrip()
+        module.deprecate(
+            'Passing a file name as the cert argument has been deprecated.  '
+            'Please use a lookup instead, see the documentation for examples.',
+            version='4.0.0', collection_name='community.aws')
     if key and os.path.isfile(key):
         with open(key, 'r') as key_fh:
             key = key_fh.read().rstrip()
+        module.deprecate(
+            'Passing a file name as the key argument has been deprecated.  '
+            'Please use a lookup instead, see the documentation for examples.',
+            version='4.0.0', collection_name='community.aws')
     if cert_chain and os.path.isfile(cert_chain):
         with open(cert_chain, 'r') as cert_chain_fh:
             cert_chain = cert_chain_fh.read()
+        module.deprecate(
+            'Passing a file name as the cert_chain argument has been deprecated.  '
+            'Please use a lookup instead, see the documentation for examples.',
+            version='4.0.0', collection_name='community.aws')
     return cert, key, cert_chain
 
 
 def main():
+
+    global module
```

Comment:
Since the module has been defined as global, I think we can remove module passed as a function parameter (`cert_action()` and `dup_ckeck()`). Please ignore this comment if you are planning to do that in an upcoming PR.
In any case, this Lgtm!

Reply:
It's coming in a later PR where I rewrite the whole thing for boto3/botocore